apps/docs/src/app/platform/component-docs/platform-slider/examples/form-field/slider-form-field-example.component.html

```diff
@@ -1,11 +1,11 @@
 <fdp-form-group class="slider-example" [formGroup]="customForm">
     <fdp-form-field id="value1" label="Single mode">
-        <fdp-slider name="value5" formControlName="value1"> </fdp-slider>
+        <fdp-slider name="value5" formControlName="value1" ariaLabel="Single mode"> </fdp-slider>
```

Comment:
should `name` also be `value1`?

Reply:
also PR title can be changed to `fix(platform): fix a11y issues for slider` in accordance with having only present tense in the message.